plugins/botsz.py

```diff
@@ -0,0 +1,50 @@
+# Professor | Albert Einstein
+# @TheUnusualPsychopath | @Albert_Einetin_TG
+# @TrojanzHEX | @CrazyBotsz
+
+import os
+import time
+import pytz
+from datetime import datetime
+from userge import userge, Message
+
+UpdatesChannel = os.environ.get("UPDATES_CHANNEL")

```

Comment:
use of this var ?

Reply:
![Uploading IMG_20210107_124848.jpg…]() That Vars Is Optional As You Can See In The Ss it Just To Mention Your Bot's Update Channel if You Wants Too..... This Plugin was developed in thinking of users who runs a bot channel..... That Vars Can Be Optional.... Just Make It Empty..... I Will Edit the Code To Add A Blank Space If People Dont Want Any Username Name Up There....

Reply:
sorry, i didn't see that :+1:

Reply:
Not a problem at all....Shall I Add The Code For Making It Blank If Someone Dont Wanna Add Such Username????

Reply:
ok, np